static/sass/_pattern_cube-progression.scss

```diff
@@ -202,4 +169,23 @@
       );
     }
   }
+
+  .is-faded {
+    // Without !important the background only applies to certain microcerts
```

Comment:
This is due to the use of `background: ...` as a style on elements such as ".p-cube-progression.is-bash".

The shorthand (`background:`) has a high speciality than (`background-color`, `background-image`, etc). It's generally good practice to use the most specific styling attribute if you are setting a single value. Because of strange issues like these.

Could you replace the use to `background:` with `background-image:` and you shouldn't need this important.

Reply:
Thanks for the explanation - very helpful